dashboard/app/controllers/levels_controller.rb

```diff
@@ -236,8 +232,10 @@ def level_params
       :level_num,
       :user,
       :dsl_text,
+      :encrypted,
       {concept_ids: []},
-      {soft_buttons: []}
+      {soft_buttons: []},
+      {examples: []}
```

Comment:
yes, leftover from implementing a similar feature (encrypted examples) for applab levels, removing...